tests/src/system/basic/WskBasicTests.scala

```diff
@@ -466,6 +466,19 @@ class WskBasicTests
             wsk.trigger.list().stdout should include(name)
     }
 
+    it should "create a trigger using property file" in withAssetCleaner(wskprops) {
+        (wp, assetHelper) =>
+            val name = "listTriggers"
+            val tmpProps = File.createTempFile("wskprops", ".tmp")
+            val env = Map("WSK_CONFIG_FILE" -> tmpProps.getAbsolutePath())
+            wsk.cli(Seq("property", "set", "--auth", wp.authKey) ++ wskprops.overrides, env = env)
+            assetHelper.withCleaner(wsk.trigger, name) {
+                (trigger, _) =>
+                    wsk.cli(Seq("-i", "trigger", "create", name), env = env)
```

Comment:
This should assert an exit code of `0` to assure the trigger has been created properly.